packages/flutter_tools/lib/src/mdns_discovery.dart

```diff
@@ -208,7 +208,74 @@ class MDnsVmServiceDiscovery {
   Future<List<MDnsVmServiceDiscoveryResult>> _pollingVmService(
     MDnsClient client, {
     String? applicationId,
-    int? deviceVmservicePort,
+    int? deviceVmServicePort,
+    String? deviceName,
+    bool ipv6 = false,
+    bool useDeviceIPAsHost = false,
+    required Duration timeout,
+    bool quitOnFind = false,
+  }) async {
```

Comment:
This diff is confusing. The only change to `_pollingVmService`'s arguments is that `deviceVmservicePort` was renamed to `deviceVmServicePort`. `_pollingVmService` and `_doPollingVmService` have the same arguments.